### ISSUE TITLE
suggestion - refining booze should always increase the worth of it

### ISSUE THREAD

**Issue by pinkamedic** [loiste-interactive/Obenseuer-Issues#2569]
just a simple thing if you have a mash that is worth around 150 OC and then you firmament it should increase in price, but in game there is always a chance that you can turn that 150 OC mash into 50 OC after fermentation, im just suggesting a simple change that when refining booze it should always add onto its worth not decrees it. it would still go off of the quality of the booze, like refining 150 OC mash but you get a shit quality refinement instead of the worth going down you just get around 165 worth booze

**Comment by Nextej** [comment]
I disagree as the process of refining alcohol is a risk for unskilled player "Do I sell it now with the current price or do I risk and try to refine it with a prospect of selling it for even more, but with risk it will be worth less". Also having this change would allow for a low-skill player to refine high tier alcohols without a need to have high skill.

What I would do is drastically reduce the deviation range of the output product and ensure that the on certain skill levels the refined alcohol will be of around the same quality (+- 0,25) as the input alcohol.
